timesketch/lib/aggregators.py

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 """Elasticsearch aggregations."""
 
+# Max number of search hits to chart
+MAX_RESULT_LIMIT = 300000
```

Comment:
Just curious; How did you come up with this limit?

Reply:
Good question, I did some simple memory monitoring to see the sweetspot for most setups. This should be moved to config though. Will add todo.